en/docs/deploy/configure-an-sp-and-idp-using-configuration-files.md

```diff
@@ -602,13 +594,12 @@ Now you can test if the configurations you have done work in a tenant scenario.
     !!! note
         If you have **not enabled** signature validation, proceed to the next step.
     
-    1.  Login using tenant credentials to the management console and
-        navigate to
-        `            Main > Manage > Keystores > List.           `
+    1.  Log in using tenant credentials to the management console and
```

Comment:
provide the management console URL and going forward I think it is better if we can have it as WSO2 Identity Server management console. WDYT?